src/NativeShell/src/RemotePlayerService.java

```diff
@@ -92,9 +89,16 @@ private void handleIntent(Intent intent) {
             switch (action) {
                 case Constants.ACTION_PLAY:
                     mediaController.getTransportControls().play();
+                    // background playback for an hour
+                    if (NativeShell.wakeLock != null && !NativeShell.wakeLock.isHeld()) {
+                        NativeShell.wakeLock.acquire(36000000);
```

Comment:
hmm, why limit to 1 hour background playback?.. seems rather arbitrary number...

Reply:
Agreed, I know I've casted for longer ;-)